specification/cognitiveservices/data-plane/TranslatorText/readme.md

```diff
@@ -0,0 +1,134 @@
+# Cognitive Services Translator Text SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Translator Text SDK.
+
+The current release is `release_3_0`.
+
+``` yaml
+
+tag: release_3_0
+add-credentials: true
+openapi-type: data-plane
+```
+
+# Releases
+
+### Release 3.0
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: stable/v3.0/TranslatorText.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-nodejs
+ 
+  
+    after_scripts:
```

Comment:
This is required only if you want Ruby SDK. Either remove after_scripts section or add azure-sdk-for-ruby above it

Reply:
`after_scripts` removed~